### PR TITLE
Rework slider on a Form page

### PR DESCRIPTION
**Pull request by Thibrac**
### Changes

ece3b0f11947cb26aa7b4864e3191fd7782398f4 : 
- Slider moving from 0.1 by 0.1
- Precise current multiplier displayed above cursor

### CODE REVIEW

File: frontend/src/components/MultiplierSelector.jsx [hunk 1]
```diff
@@ -5,25 +5,24 @@ import './multiplier.css';
...
   const { data, isLoading, error } = useMaxMultiplier();
-  const [actualValue, setActualValue] = useState(1.0);
+  const [actualValue, setActualValue] = useState(0.0);
   const sliderRef = useRef(null);
   const isDragging = useRef(false);
-
-  const maxMultiplier = useMemo(() => {
-  return Math.round(parseFloat((data?.[selectedToken]))) || 5.0;
-}, [data, selectedToken]);
+  const maxMultiplier = useMemo(() => 10, [data, selectedToken]);
```
Comment (whateverfw) [inline frontend/src/components/MultiplierSelector.jsx:11]: it should be not hardcoded to 10, rather it should be equal to a value returned from a backend

File: frontend/src/components/multiplier.css
Comment (whateverfw) [inline frontend/src/components/multiplier.css:1]: there is not blue tooltip like in figma, please fix

**Review by whateverfw — CHANGES_REQUESTED** [review]
@Thibrac can you please also change condition in `Form.jsx` 48 line ` if (!positionData?.has_opened_position) {` to be `if (positionData?.has_opened_position)`

File: frontend/src/components/multiplier.css
Comment (whateverfw) [inline frontend/src/components/multiplier.css:1]: point css is not equal to figma's

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — CHANGES_REQUESTED** [review]
Slider thickness is incorrect on mobile screens, as well as horizontal and vertical margins. 
Please do exactly like in Figma

**Comment by whateverfw** [comment]
@Thibrac Just in case: css is not fixed in last commit

**Comment by whateverfw** [comment]
@Thibrac I don't think it will pass in front of "Select multiplier" if you do everything according to Figma, because there is padding from top
image

File: frontend/src/components/multiplier.css
Comment (whateverfw) [reply frontend/src/components/multiplier.css:1]: there is no triangle at the bottom of a tooltip

Comment (whateverfw) [reply frontend/src/components/multiplier.css:1]: still don't correspond to figma

Comment (whateverfw) [inline frontend/src/components/multiplier.css:1]: Slider thickness is incorrect on mobile screens, as well as horizontal and vertical margins.
Please do exactly like in Figma

Comment (whateverfw) [inline frontend/src/components/multiplier.css:1]: slider horizonal margins are not as in figma

Comment (whateverfw) [inline frontend/src/components/multiplier.css:1]: slider bottom padding are not as in figma on **mobile**

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/components/multiplier.css
Comment (Thibrac) [reply frontend/src/components/multiplier.css:1]: wip

**Comment by whateverfw** [comment]
@Thibrac tooltip is not the same as in figma

**Comment by whateverfw** [comment]
> > @Thibrac Please test before you commit and request review next time:
> > broken tooltip and padding didn't change image
> 
> Sorry, I've done some tests, on which mobile is this bug? I don't have it..

Just try to make your screen size smaller in browser, it happend for me for the screens less than 500px or smth

**Comment by Thibrac** [comment]
> > > @Thibrac Please test before you commit and request review next time:
> > > broken tooltip and padding didn't change image
> > 
> > 
> > Sorry, I've done some tests, on which mobile is this bug? I don't have it..
> 
> Just try to make your screen size smaller in browser, it happend for me for the screens less than 500px or smth

I don't have this bug with all mobile and full screen sizes... I fix the pointer under the tooltip

**Comment by whateverfw** [comment]
@Thibrac  image

Still broken, starting from 390 (Iphone 12 Pro example) till 768

**Review by whateverfw — CHANGES_REQUESTED** [review]
⏫

**Comment by Thibrac** [comment]
> @Thibrac image
> 
> Still broken, starting from 390 (Iphone 12 Pro example) till 768

Sorry but i don't have this problem on Chrome, I'll try it on another browser.

Capture d’écran 2024-11-28 221036

**Comment by whateverfw** [comment]
@Thibrac Not sure, I got this on chrome, anyways, If you will make this tooltip of size just line in figma 36x32 (which is not correctly in the app) and also consider paddings -  it should fit

**Review by whateverfw — CHANGES_REQUESTED** [review]
@Thibrac tooltip size stil doesn't correspond to figma.  (that's why it keeps breaking into multiple lines)

**Comment by Thibrac** [comment]
> @Thibrac tooltip size stil doesn't correspond to figma. (that's why it keeps breaking into multiple lines)

I'm fixing this sorry for the time

**Review by whateverfw — APPROVED** [review]

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**